index.bs

```diff
@@ -240,7 +281,7 @@ The GravitySensor Interface {#gravitysensor-interface}
 --------------------------------
 
 <pre class="idl">
-  [Constructor(optional SensorOptions options), SecureContext, Exposed=Window]
+  [Constructor(optional SpatialSensorOptions options), SecureContext, Exposed=Window]
```

Comment:
Please update also the abstract operation for the GravitySensor construction:
[=construct spatial sensor object=]

Reply:
done